ui/app/components/pending-tx.js

```diff
@@ -29,19 +29,40 @@ const sectionDivider = h('div', {
   style: {
     height: '1px',
     background: '#E7E7E7',
+    margin: 0,
+    marginTop: '18px',
   },
 })
 
 const contentDivider = h('div', {
```

Comment:
I have mostly refactored `contentDivider` out of tx-list. Since this is purely presentational, we can use the `+` or `nth-of-child` selector in css to create divider. I would not create a dedicate `div` as divider unless it serves other purpose (e.g. drag-and-drop target, click event, etc)

Reply:
I've removed contentDivider and sectionDivider